### ISSUE TITLE
TypeError: Cannot read property 'start' of null

### ISSUE THREAD

**Issue by steve-taylor** [rollup/rollup#2381]
- Rollup Version (if applicable): 0.63.4
- Operating System (or Browser): macOS High Sierra
- Node Version: 10.5.0
- NPM Version: 6.1.0

### How Do We Reproduce?

```sh
git clone https://github.com/finalforum/finalforum
cd finalforum
git checkout 9d2bb38428798b61efe80a05febee9d4e93cb549
npm ci
npm run build-client
```

### Expected Behavior

It rolls up my code into `dist/client-bundle.js`.

### Actual Behavior

It produces the following error:

```
[!] TypeError: Cannot read property 'start' of null
TypeError: Cannot read property 'start' of null
    at ConditionalExpression.render (/path/to/finalforum/finalforum/node_modules/rollup/dist/rollup.js:10536:53)
    at BinaryExpression.NodeBase.render (/path/to/finalforum/finalforum/node_modules/rollup/dist/rollup.js:9240:23)
    at CallExpression.NodeBase.render (/path/to/finalforum/finalforum/node_modules/rollup/dist/rollup.js:9236:31)
    at CallExpression.NodeBase.render (/path/to/finalforum/finalforum/node_modules/rollup/dist/rollup.js:9236:31)
    at VariableDeclarator.NodeBase.render (/path/to/finalforum/finalforum/node_modules/rollup/dist/rollup.js:9240:23)
    at VariableDeclaration$$1.render (/path/to/finalforum/finalforum/node_modules/rollup/dist/rollup.js:12375:28)
    at renderStatementList (/path/to/finalforum/finalforum/node_modules/rollup/dist/rollup.js:3836:25)
    at BlockStatement$$1.render (/path/to/finalforum/finalforum/node_modules/rollup/dist/rollup.js:9977:13)
    at IfStatement.render (/path/to/finalforum/finalforum/node_modules/rollup/dist/rollup.js:10839:33)
    at renderStatementList (/path/to/finalforum/finalforum/node_modules/rollup/dist/rollup.js:3836:25)
```

which seems to be related to #2134 and fixed in #2146 (0.58.2).

**Comment by steve-taylor** [comment]
Downgrading `styled-components` from 3.4.1 to 3.3.3 fixes it.

When I upgrade from 3.3.3 to 3.4.0, I get this new warning:

```
(!) Missing shims for Node.js built-ins
Creating a browser bundle that depends on 'stream'. You might need to include https://www.npmjs.com/package/rollup-plugin-node-builtins
(!) Unresolved dependencies
https://github.com/rollup/rollup/wiki/Troubleshooting#treating-module-as-external-dependency
stream (imported by node_modules/styled-components/dist/styled-components.browser.esm.js)
(!) Missing global variable name
Use output.globals to specify browser global variable names corresponding to external modules
stream (guessing 'stream')
```

I added `rollup-plugin-node-globals` and `rollup-plugin-node-builtins` (in that order) as the last two plugins and the warning went away.

I then upgraded styled-components from 3.4.0 to 3.4.1 and the error returned.

So I think this is an issue with both styled-components 3.4.1 and Rollup. The issue with Rollup is a lack of useful error messaging in this case.

**Comment by lukastaegert** [comment]
@steve-taylor Thanks for this issue! I have been able to boil it down to this minimal repro:
https://rollupjs.org/repl?version=0.63.5&shareable=JTdCJTIybW9kdWxlcyUyMiUzQSU1QiU3QiUyMm5hbWUlMjIlM0ElMjJtYWluLmpzJTIyJTJDJTIyY29kZSUyMiUzQSUyMmlmJTIwKHRydWUpJTIwJTdCJTVDbiU1Q3R2YXIlMjB4JTIwJTNEJTIwMCUzQiU1Q24lN0QlMjBlbHNlJTIwJTdCJTVDbiU1Q3R2YXIlMjB4JTIwJTNEJTIwdW5rbm93biUyMCUzRiUyMDMlMjAlM0ElMjA0JTNCJTVDbiU3RCU1Q24lNUNuY29uc29sZS5sb2coeCklM0IlMjIlN0QlNUQlMkMlMjJvcHRpb25zJTIyJTNBJTdCJTIyZm9ybWF0JTIyJTNBJTIyY2pzJTIyJTJDJTIybmFtZSUyMiUzQSUyMm15QnVuZGxlJTIyJTJDJTIyZ2xvYmFscyUyMiUzQSU3QiU3RCUyQyUyMmFtZCUyMiUzQSU3QiUyMmlkJTIyJTNBJTIyJTIyJTdEJTdEJTJDJTIyZXhhbXBsZSUyMiUzQW51bGwlN0Q=

I see what I can do to fix it.

**Comment by lukastaegert** [comment]
Fix at #2384
